src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -132,8 +132,8 @@ limitations under the License.
 
   <struct name="ContentLaunchParamater">
     <cluster code="0x050a"/>
-    <item name="Type" type="ContentLaunchParameterEnum"/>
+    <item name="Type" type="ContentLaunchParameterEnum" enum="true"/>
     <item name="Value" type="CHAR_STRING"/>
-    <item name="ExternalIDList" type="ARRAY" entryType="ContentLaunchAdditionalInfo" length="254"/>
+    <item name="ExternalIDList" type="ContentLaunchAdditionalInfo" array="true" length="254"/>
```

Comment:
Also not sure why we want this change, or why we have two different ways of representing arrays in XML.  @vivien-apple ?

Reply:
in @tecimovic change, for array item in struct, it remove entryType and use array=true to do that

Reply:
> Also not sure why we want this change, or why we have two different ways of representing arrays in XML. @vivien-apple ?

A recent ZAP change has removed `entryType` for struct item. It probably looks better to use `array=true` and to use `type=something` instead of `type= `ARRAY` and `entryType=something`. But I agree that we should update ZAP to have a single way to declare it :(

Reply:
@tecimovic Do you have some "free" time to help unifying things here ?

Reply:
agree. @vivien-apple , we should unify this. entryType is still used in other place, which makes me confused as well and thought this would break chip when I chat with you on entrytype/isArray yesterday in slack

Reply:
https://github.com/project-chip/zap/issues/239